From 48fa21720920ef97a87ff5ae5ff8262c7ea9189f Mon Sep 17 00:00:00 2001
From: David Sniatkiwicz <davidsn@marvell.com>
Date: Sun, 18 Mar 2018 14:34:45 +0200
Subject: [PATCH 0579/1051] dts: a8kp: rename a8080 setup to pd instead of db

Change-Id: I025e20c4e0615102a3c5fcf149b38bc2c7767b84
Signed-off-by: David Sniatkiwicz <davidsn@marvell.com>
[Kevin: The original patch got from Marvell sdk10.0_19.06.
Just some minor context mods in order to port to wrlinux]
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 Documentation/devicetree/bindings/arm/marvell/armada-8kp.txt    | 2 +-
 arch/arm64/boot/dts/marvell/Makefile                            | 2 +-
 .../boot/dts/marvell/{armada-8080-db.dts => armada-8080-pd.dts} | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)
 rename arch/arm64/boot/dts/marvell/{armada-8080-db.dts => armada-8080-pd.dts} (88%)

diff --git a/Documentation/devicetree/bindings/arm/marvell/armada-8kp.txt b/Documentation/devicetree/bindings/arm/marvell/armada-8kp.txt
index f3e9624534c6..e290912d16e8 100644
--- a/Documentation/devicetree/bindings/arm/marvell/armada-8kp.txt
+++ b/Documentation/devicetree/bindings/arm/marvell/armada-8kp.txt
@@ -11,5 +11,5 @@ the following root node property:
 
 Example:
 
-compatible = "marvell,armada-8080-db", "marvell,armada-8080",
+compatible = "marvell,armada-8080-pd", "marvell,armada-8080",
 	     "marvell,armada-ap810-octa", "marvell,armada-ap810"
diff --git a/arch/arm64/boot/dts/marvell/Makefile b/arch/arm64/boot/dts/marvell/Makefile
index 81d8d8ac0c3a..1485a1e569be 100644
--- a/arch/arm64/boot/dts/marvell/Makefile
+++ b/arch/arm64/boot/dts/marvell/Makefile
@@ -6,4 +6,4 @@ dtb-$(CONFIG_ARCH_MVEBU) += armada-7040-db.dtb
 dtb-$(CONFIG_ARCH_MVEBU) += armada-8040-db.dtb
 dtb-$(CONFIG_ARCH_MVEBU) += armada-8040-mcbin.dtb
 dtb-$(CONFIG_ARCH_MVEBU) += armada-8040-mcbin-single-shot.dtb
-dtb-$(CONFIG_ARCH_MVEBU) += armada-8080-db.dtb
+dtb-$(CONFIG_ARCH_MVEBU) += armada-8080-pd.dtb
diff --git a/arch/arm64/boot/dts/marvell/armada-8080-db.dts b/arch/arm64/boot/dts/marvell/armada-8080-pd.dts
similarity index 88%
rename from arch/arm64/boot/dts/marvell/armada-8080-db.dts
rename to arch/arm64/boot/dts/marvell/armada-8080-pd.dts
index 4ba158f415ce..a41d74ffc6fd 100644
--- a/arch/arm64/boot/dts/marvell/armada-8080-db.dts
+++ b/arch/arm64/boot/dts/marvell/armada-8080-pd.dts
@@ -9,7 +9,7 @@
 
 / {
 	model = "Marvell 8080 board";
-	compatible = "marvell,armada-8080-db", "marvell,armada-8080",
+	compatible = "marvell,armada-8080-pd", "marvell,armada-8080",
 		     "marvell,armada-ap810-octa", "marvell,armada-ap810";
 
 	chosen {
-- 
2.17.1

